charts/pega/templates/_pega-tomcat-keystore-secret.tpl

```diff
@@ -0,0 +1,32 @@
+{{- define "pegaTomcatKeystoreSecretTemplate" }}
+# Secret used for tls certificates to configure https to tomcat
+{{- if .node.tls }}
+{{- if .node.tls.enabled }}
+kind: Secret
+apiVersion: v1
+type: Opaque
+metadata:
+  name: {{ .name }}-tomcat-keystore-secret
+  namespace: {{ .root.Release.Namespace }}
+stringData:
+  # cert Files
+  # Base64 encoded password for enabling TLS in tomcat
+{{- if .node.tls.keystorepassword }}
+  TOMCAT_KEYSTORE_PASSWORD: {{ .node.tls.keystorepassword | quote}}
+{{- else }}
+  TOMCAT_KEYSTORE_PASSWORD: "123456"
```

Comment:
`changeme` or `changeit` would be a better one? unless it takes a lot to re-generate the jks file